### PR TITLE
Updated MP3 Driver with busy and MP3 Folder command

### PR DESCRIPTION
**Pull request by trlafleur**
## Description: Updated MP3 Driver with busy and MP3 Folder command

**Related issue (if applicable):** fixes #<Tasmota issue number goes here>

## Checklist:
  - [ x] The pull request is done against the latest development branch
  - [ x] Only relevant files were touched
  - [ x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [ ] The code change is tested and works with Tasmota core ESP8266 V.2.7.4.9
  - [x ] The code change is tested and works with Tasmota core ESP32 V.2.0.3
  - [ x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

**Comment by Jason2866** [comment]
Please add the changes to all language files.

File: tasmota/include/tasmota_template.h [hunk 1]
```diff
@@ -54,6 +54,7 @@ enum UserSelectablePins {
   GPIO_SDM630_TX, GPIO_SDM630_RX,      // SDM630 Serial interface
   GPIO_TM1638CLK, GPIO_TM1638DIO, GPIO_TM1638STB,  // TM1638 interface
   GPIO_MP3_DFR562,                     // RB-DFR-562, DFPlayer Mini MP3 Player
+  GPIO_MP3_DFR562_BUSY,                // RB-DFR-562, DFPlayer Mini MP3 Player busy flag
```
Comment (arendst) [inline tasmota/include/tasmota_template.h:57]: As stated at the top of the file only append this list. never insert as it breaks all devices.

File: tasmota/include/tasmota_template.h [hunk 2]
```diff
@@ -286,7 +287,7 @@ const char kSensorNames[] PROGMEM =
   D_SENSOR_SDM120_TX "|" D_SENSOR_SDM120_RX "|"
   D_SENSOR_SDM630_TX "|" D_SENSOR_SDM630_RX "|"
   D_SENSOR_TM1638_CLK "|" D_SENSOR_TM1638_DIO "|" D_SENSOR_TM1638_STB "|"
-  D_SENSOR_DFR562 "|"
+  D_SENSOR_DFR562 "|" D_SENSOR_DFR562_BUSY "|"
```
Comment (arendst) [inline tasmota/include/tasmota_template.h:290]: Same here.

File: tasmota/language/pl_PL.h [hunk 3]
```diff
@@ -631,7 +631,8 @@
 #define D_SENSOR_I2S_IN_CLK    "I2S In Clk"
 #define D_SENSOR_I2S_IN_SLCT   "I2S In Slct"
 #define D_SENSOR_WS2812        "WS2812"
-#define D_SENSOR_DFR562        "Odtwarzacz MP3"
+#define D_SENSOR_DFR562        "MP3 Odtwarzacz"
```
Comment (arendst) [inline tasmota/language/pl_PL.h:634]: If your polish is as bad as mine then do not change already translated text

File: tasmota/tasmota_xdrv_driver/xdrv_14_mp3.ino [hunk 4]
```diff
@@ -326,14 +399,24 @@ bool Xdrv14(uint8_t function)
...
         MP3PlayerInit();                              // init and start communication
         break;
+
       case FUNC_COMMAND:
         result = MP3PlayerCmd();                      // return result from mp3 player command
         break;
+
+      case FUNC_EVERY_SECOND:
+
+        TasmotaGlobal.seriallog_timer = 600;          // DEBUG Tool   <---------- TRL
```
Comment (arendst) [inline tasmota/tasmota_xdrv_driver/xdrv_14_mp3.ino:416]: Should not be here. Pls remove.

File: tasmota/tasmota_xdrv_driver/xdrv_14_mp3.ino [hunk 5]
```diff
@@ -19,6 +19,14 @@
   --------------------------------------------------------------------------------------------
   Version yyyymmdd  Action    Description
   --------------------------------------------------------------------------------------------
+
+  1.0.0.6 22Jun2022 added   - Busy flag on optional GPIO pin
```
Comment (arendst) [inline tasmota/tasmota_xdrv_driver/xdrv_14_mp3.ino:23]: Keep date format consistend with Tasmota: 20220622

**Pull request merged**